FastBank.Services/BankAccountService/BankAccountService.cs

```diff
@@ -105,9 +110,113 @@ public void WithdrawAmount(Customer customer, BankAccount customerBankAccount)
             }
         }
 
-        public void TransferAmountToFriend(Customer customer, BankAccount customerBankAccount, Customer friend, decimal amount)
+        public void TransferMoneyToFriend(Customer customer, BankAccount customerBankAccount, Dictionary<int, User> friends)
```

Comment:
Is it possible to put Customer and BankAccount in one domain class? 
CustomerBankAccount for example